pylint/checkers/base.py

```diff
@@ -142,17 +142,24 @@ def _is_multi_naming_match(match, node_type, confidence):
 
 
 if sys.version_info < (3, 0):
-    PROPERTY_CLASSES = set(('__builtin__.property', 'abc.abstractproperty'))
+    BUILTIN_PROPERTY = '__builtin__.property'
 else:
-    PROPERTY_CLASSES = set(('builtins.property', 'abc.abstractproperty'))
+    BUILTIN_PROPERTY = 'builtins.property'
 
 
-def _determine_function_name_type(node):
+def _determine_function_name_type(node, config=None):
     """Determine the name type whose regex the a function's name should match.
 
     :param node: A function node.
+    :param config: Configuration from which to pull additional property classes.
     :returns: One of ('function', 'method', 'attr')
     """
+    property_classes = set((BUILTIN_PROPERTY,))
```

Comment:
Can you move this small block into a private function, which returns a tuple of property classes and names?